components/linkedin/actions/create-text-post-user/create-text-post-user.mjs

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+export default {
+  key: "linkedin_create-text-post-user",
```

Comment:
You can use `linkedin-` instead of an underscore. We use underscores to separate spaces in multi-word app names (like `google_calendar`), but dashes to separate app name + component name. See https://pipedream.com/docs/components/guidelines/#component-key-pattern

Reply:
changed!